### ISSUE TITLE
!! mlrpt: Failed to parse config file!

### ISSUE THREAD

**Issue by benb0jangles** [dvdesolve/mlrpt#3]
Hi there, 
I try to install mlrpt on MacOS

I can run command:
`mlrpt -h`

result:
```
Usage:  mlrpt -[c <config-file> s <HHMM-HHMM> -qihv]
       -c: configuration file
       -s: start and stop operation time in HHMM format
       -q: run in quiet mode (no messages printed)
       -i: flip images (useful for South to North passes)
       -h: print this usage information and exit
       -v: print version information and exit

```
Also:
`mlrpt -v`

result:
`mlrpt 1.8.0`

However I get error with

command:
`mlrpt -c meteor-m2.cfg`

error:
```
!! mlrpt: Failed to parse config file!
(null):0 - file I/O error

!! mlrpt: Failed to load mlrpt configuration - exiting
```

Any ideas? Thanks

**Comment by dvdesolve** [comment]
Could you please check the same config file on another machine/OS?

**Comment by benb0jangles** [comment]
Solved using:
cd /Users/username/mlrpt/share/config
mlrpt -c meteor-m2.cfg

I just had to find the config directory :)

Working :
iMac:config benji$ mlrpt -c Meteor-M2.cfg
mlrpt: Will use Auto-Detected SDR Device
mlrpt: Device has Frequency Correction
mlrpt: Set Auto Gain Control
mlrpt: Setting up Receive Stream
[INFO] Using format CS16.
mlrpt: Receive Stream set up OK
mlrpt: SoapySDR Initialized OK
mlrpt: Decoding from SoapySDR Receiver
mlrpt: Decode Timer Started: 900 sec
mlrpt: Decoding of LRPT Images Started
mlrpt: Receive Stream activated OK
mlrpt: PLL Unlocked
mlrpt: PLL Locked

So now I just have to rebuild SoapySDR to include Airspy & rtl-sdr dongle. My RSP1 is no good for meteor receiving.
